core/domain/suggestion_services.py

```diff
@@ -593,12 +619,199 @@ def get_translation_suggestions_waiting_longest_for_review_per_lang(
     return [
         get_suggestion_from_model(suggestion_model) for suggestion_model in (
             suggestion_models.GeneralSuggestionModel
-            .get_translation_suggestions_waiting_longest_for_review_per_lang(
+            .get_translation_suggestions_waiting_longest_for_review(
                 language_code)
         )
     ]
 
 
+def _get_plain_text_from_html_content_string(html_content_string):
+    """Retrieves the plain text from the given html content string. RTE element
+    occurrences in the html are replaced by their corresponding name,
+    capitalized in square brackets.
+    eg: <p>Sample1 <oppia-noninteractive-math></oppia-noninteractive-math>
+        Sample2 </p> will give as output: Sample1 [MATH] Sample2.
+    Note: similar logic exists in the frontend in format-rte-preview.filter.ts.
+
+    Args:
+        html_content_string: str. The content html string to convert to plain
+            text.
+
+    Returns:
+        str. The plain text string from the given html content string.
+    """
+
+    def _replace_rte_tag(rte_tag):
+        """Replaces all of the <oppia-noninteractive-**> tags.
+
+        Args:
+            rte_tag: MatchObject. The oppia-noninteractive rte tag.
```

Comment:
```suggestion
            rte_tag: MatchObject. The matched oppia-noninteractive rte tag.
```

This is slightly confusing. This parameter seems to really be a representation of the math, not the actual tag.

Reply:
It should be the tag - what makes you say that it is not?

Reply:
Sorry, what I mean is that the type is 'MatchObject' not string, so it's not exactly the 'tag', right? It's more like an object that be used to retrieve the tag, unless I'm misunderstanding something.